### PR TITLE
[TT-6363] Update security vulnerability

### PR DESCRIPTION
**Pull request by c5haw**
- Update high security vulnerability `axios` to latest
- Add `gts`

### CODE REVIEW

File: src/version.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const version = '3.3.3';
\ No newline at end of file
+export const version = '3.3.3';
```
Comment (downeyfe) [inline src/version.ts:1]: Should this be updated to 3.3.4?

Comment (c5haw) [reply src/version.ts:1]: 🙄  yes, this is another project with the annoyance of having multiple places to look for bumping versions which is really redundant.

Comment (downeyfe) [reply src/version.ts:1]: Agreed, what is this even used for? 😥

**Review by downeyfe — APPROVED** [review]

File: src/version.ts (hunk 1 above)
Comment (c5haw) [reply src/version.ts]: This was the published `package.json` and also referred to for the version which is put into the version file.

The next iteration of this (which will be a breaking change) I want to improve this repository and stop supporting both browser fetch as well as node axios. The project is meant to be for purely node wrapper and not browser support. The javascript wrapper does that.

Comment (downeyfe) [reply src/version.ts]: Sounds like a plan!

**Pull request merged**